schemas/dc-library.xsd

```diff
@@ -1,389 +1,400 @@
-      <xs:attributeGroup ref="codify:allAttributes"/>
-    </xs:complexType>
-  </xs:element>
-  <xs:group name="sectionContents">
-    <xs:sequence>
-      <xs:element ref="heading" minOccurs="0"/>
-      <xs:element ref="text" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:element ref="toc" minOccurs="0"/>
-      <xs:element ref="include" minOccurs="0"/>
-      <xs:element name="container" type="sectionContainer" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:element ref="para" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:element ref="aftertext" minOccurs="0" maxOccurs="unbounded"/>
-      <xs:element ref="annotations" minOccurs="0"/>
-      <xs:element ref="annotation" minOccurs="0" maxOccurs="unbounded"/>
-    </xs:sequence>
-  </xs:group>
-  <xs:element name="text">
-    <xs:complexType>
-      <xs:complexContent mixed="true">
-        <xs:extension base="xs:anyType">
-          <xs:attribute ref="codify:value" use="optional"/>
-        </xs:extension>
-      </xs:complexContent>
+            <xs:element ref="heading"/>
+            <xs:choice maxOccurs="unbounded">
+                <xs:element name="container" type="tocContainer"/>
+                <xs:element name="section" type="tocSection"/>
+            </xs:choice>
+        </xs:sequence>
     </xs:complexType>
-  </xs:element>
-  <xs:element name="toc">
-    <xs:complexType>
-      <xs:sequence>
-        <xs:choice maxOccurs="unbounded">
-          <xs:element ref="text"/>
-          <xs:element name="container" type="tocContainer"/>
-          <xs:element name="section" type="tocSection"/>
-        </xs:choice>
-      </xs:sequence>
-      <xs:attribute name="containing-doc" type="xs:string" use="optional"/>
-      <xs:attributeGroup ref="codify:allAttributes"/>
+    <xs:complexType name="tocSection">
+        <xs:sequence>
+            <xs:element ref="num"/>
+            <xs:element ref="heading"/>
+        </xs:sequence>
     </xs:complexType>
-  </xs:element>
-  <xs:complexType name="tocContainer">
-    <xs:sequence>
-      <xs:element ref="prefix" minOccurs="0"/>
-      <xs:element ref="num"/>
-      <xs:element ref="heading"/>
-      <xs:choice maxOccurs="unbounded">
-        <xs:element name="container" type="tocContainer"/>
-        <xs:element name="section" type="tocSection"/>
-      </xs:choice>
-    </xs:sequence>
-  </xs:complexType>
-  <xs:complexType name="tocSection">
-    <xs:sequence>
-      <xs:element ref="num"/>
-      <xs:element ref="heading"/>
-    </xs:sequence>
-  </xs:complexType>
-  <xs:attributeGroup name="targetAttributes">
-    <xs:attribute name="doc" type="xs:string" use="optional"/>
-    <xs:attribute name="path" type="xs:string" use="optional"/>
-  </xs:attributeGroup>
+    <!-- attribute groups -->
+    <xs:attributeGroup name="targetAttributes">
+        <xs:attribute name="doc" type="xs:string" use="optional"/>
+        <xs:attribute name="path" type="xs:string" use="optional"/>
+    </xs:attributeGroup>
 </xs:schema>
```

Comment:
everything should be indented with 2 spaces. can't see what was changed in the dc-library xsd

Reply:
We currently have the spacing defaulting to 4 in our configuration:
```
 "configurationDefaults": {
      "[xml]": {
        "editor.wordWrap": "on",
        "editor.tabSize": 4
      }
    },
```
If this needs to be changed, I can, though.